litmus-portal/frontend/package.json

```diff
@@ -38,7 +38,7 @@
     "jsonwebtoken": "^8.5.1",
     "jspdf": "^2.1.1",
     "jspdf-autotable": "^3.5.13",
-    "litmus-ui": "1.3.1",
+    "litmus-ui": "^1.4.1",
     "localforage": "^1.7.3",
```

Comment:
Please mak make it `litmus-ui: 1.4.0`. the version 1.4.1 has some issues

Reply:
Made the change already can you once again take a look 😅